pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1473,6 +1473,10 @@ type ObjectStoreSpec struct {
 	// +nullable
 	Gateway GatewaySpec `json:"gateway"`
 
+	// Enable RGW Ops Logs for S3 in a sidecar
+	// +optional
+	RGWOpsLogEnabled bool `json:"RGWOpsLogEnabled,omitempty"`
```

Comment:
Based on latest Go/k8s best practices for naming, let's make a minor change here

```suggestion
	RgwOpsLogEnabled bool `json:"rgwOpsLogEnabled,omitempty"`
```